### DISCUSSION TITLE
How do I copy a configuration from one stream deck to another?

### DISCUSSION THREAD

**Discussion by sallymanda** [streamdeck-linux-gui/streamdeck-linux-gui#127] [Q&A]
Heyy :-) I have three stream decks in different locations which will, mostly, have the same button layouts, I'm just wondering how to copy the button configuration from my first stream deck to my others without having to manually reconfigure each one separately?

**Comment by coolapso** [comment]
since [v4.0.0](https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/releases/tag/v4.0.0) there's default config per model, this means that if all streamdecks with same model will share the same config if they don't have any configuration for that specific serial number.

**Comment by Danie10** [comment]
Export and import menu options should have worked?

**Comment by sallymanda** [comment]
Hey thanks for responses :-) It seems like streamdeck is identifying the two devices as different models maybe? I'm not sure if the number listed next to the device is the serial number or the model number. The two minis are listed as `Stream Deck Mini - BL19K1B01105` and `Stream Deck Mini - BL25K1B02780`.

I tried export and import but I get:

```
libusb: debug [libusb_submit_transfer] transfer 0x352b970
Segmentation fault (core dumped)
```

But I think the import/export is for the whole application, not for the specific unit after looking at the JSON file. I tried copy pasting the contents of the JSON file for the first device into the section for second device, but it crashed, not familar with JSON so probably some obvious mistakes, I can have a closer look at it later in the week if I find the time.
